### ISSUE TITLE
car-back

### ISSUE THREAD

**Issue by Leinad4Mind** [Templarian/MaterialDesign#4019]
We've the car front icon but not the car rear, would be great to have it.

As an idea, the car rear could be similar to the front but with a different size. 

And the front could be enhance by adding the steering like this: ![Image](https://i.imgur.com/JnpDaJR.jpg)

**Comment by goyney** [comment]
Paging @MrGrigri. He created this icon a while back, but didn't add it to the library.

**Comment by MrGrigri** [comment]
I created this about three weeks ago and we decided internally that it wasn't necessary to add. Here is the SVG in case we decide to add it. The shadow path is the current `car` icon. I raised the tail clearance and changed the shape of the glass. I also added a license plate and square tail lights to help distinguish between the two.

From a UX perspective, since these two are so simmilar, they need to have supporting text to help the user and shouldn't stand alone.

[![image](https://user-images.githubusercontent.com/4528551/54213266-c104f480-44a9-11e9-9082-eba3ca09454e.png)](https://github.com/Templarian/MaterialDesign/files/2957547/car-back.zip)

![image](https://user-images.githubusercontent.com/4528551/54213508-1c36e700-44aa-11e9-889c-8fa97173bd9f.png)
